ci/redmine_bugzilla.py

```diff
@@ -38,6 +38,7 @@
 from bugzilla.rhbugzilla import RHBugzilla
 from redmine import Redmine
 import urllib3.contrib.pyopenssl
+import xmlrpclib
```

Comment:
This dep probably also needs to be installed [here](https://github.com/pulp/pulp_packaging/blob/master/ci/jobs/redmine.yaml#L40) right?